.gitignore

```diff
@@ -2,3 +2,4 @@
 .ruby-version
 *.gem
 Gemfile.lock
+.idea
```

Comment:
✂️ You should add this to your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) so you don't have to add it to every project!